tests/sklearn/test_sklearn_models.py

```diff
@@ -626,8 +642,8 @@ def check_pipeline(model_class, x, y):
         param_grid = {
             "model__n_bits": [2, 3],
         }
-
-    grid_search = GridSearchCV(pipe_cv, param_grid, error_score="raise", cv=3)
+    # Since the data-set is really small for KNN, we have to decrease the number of splits
```

Comment:
this comment is not necessary I think, you could remove it (or instead say "we need a small number of splits")